data/src/main/java/com/antosik/benchproject/data/movie/details/entity/GenresEntity.kt

```diff
@@ -0,0 +1,15 @@
+package com.antosik.benchproject.data.movie.details.entity
+
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+import com.antosik.benchproject.domain.movie.details.entity.GenresModel
+
+@Entity
+internal data class GenresEntity(
+    @PrimaryKey
+    val genresId: Int,
```

Comment:
Minor: Could be in one line `@PrimaryKey val genresId: Int`

Reply:
I changed it